src/dialogs/permissions.tsx

```diff
@@ -258,9 +275,9 @@ const EditPermissionsModal = ({ dialogResult, selected, path }) => {
           variant={ModalVariant.small}
           /* Translators: $0 represents a filename */
           title={fmt_to_fragments(_("$0 permissions"), <b>{selected.name}</b>)}
-          description={inode_types[selected.type] || "Unknown type"}
+          description={(selected.type) ? inode_types[selected.type] : "Unknown type"}
```

Comment:
This one I don't understand? Also easy test coverage fix :D And hmm even worse missing translations this should be:

`_("Missing type")` woops!

Reply:
it doesn't really like that `selected.type` can be `undefined` when directly using it as index type. And yeah, translation! 😮